src/node_file.cc

```diff
@@ -776,7 +815,9 @@ void Access(const FunctionCallbackInfo<Value>& args) {
   } else {  // access(path, mode, undefined, ctx)
     CHECK_EQ(argc, 4);
     fs_req_wrap req_wrap;
+    FS_SYNC_TRACE_BEGIN(SYNC_TRACE_ACCESS);
     SyncCall(env, args[3], &req_wrap, "access", uv_fs_access, *path, mode);
```

Comment:
Maybe pass this as a parameter to `SyncCall` and call `FS_SYNC_TRACE_BEGIN` and `FS_SYNC_TRACE_END` from there?

Reply:
Thanks very much for the comment. I also considered adding trace begin and end into SyncCall. These trace functions are flexible to take additional useful tracing data whenever making sense such as number of bytes read for read(). Since SyncCall is already a variadic function, it will be a little messy to introduce arbitrary trace arguments there. It seems cleaner to leave SyncCall API unchanged and call trace begin and end variadic functions separately.

Reply:
@chinhuang007 Good point, I missed the call below that passes additional arguments to the macros, sorry about that.